Library/Homebrew/test/Rakefile

```diff
@@ -1,8 +1,14 @@
 require "rake"
 require "rake/testtask"
 
+def mac?
+  /darwin/i === RUBY_PLATFORM && !ENV["HOMEBREW_TEST_GENERIC_OS"]
```

Comment:
I copy-pasted from `os.rb`. Update in both? I also thing `RUBY_PLATFORM.include?("darwin")` is sufficient.